src/chain.h

```diff
@@ -324,6 +324,11 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(VARINT(nDataPos));
         if (nStatus & BLOCK_HAVE_UNDO)
             READWRITE(VARINT(nUndoPos));
+        uint8_t tag = 0;
```

Comment:
If the next release resets the testnet, we won't need format compatibility. Likewise if it starts the mainnet.